library/network_connections.py

```diff
@@ -1988,6 +2015,7 @@ def _check_ethtool_setting_support(self, idx, connection):
                     idx, "ethtool.%s specified but not supported by NM", specified
                 )
 
+            # pylint: disable=blacklisted-name
```

Comment:
see the other comment about this change

Reply:
I rewrote the code to avoid this issue e.g. use `for option in specified.keys()`